ide-support/lsp/lsp-server/src/main/java/io/helidon/lsp/server/core/LanguageServerContext.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2022 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.lsp.server.core;
+
+import java.util.List;
+
+import org.eclipse.lsp4j.WorkspaceFolder;
+
+/**
+ * Context data for the running instance of the Helidon Language Server.
+ */
+public class LanguageServerContext {
+
+    private List<WorkspaceFolder> workspaceFolders;
+
+    /**
+     * Get workspace folders in IDE.
+     *
+     * @return List of workspace folders.
+     */
+    public List<WorkspaceFolder> getWorkspaceFolders() {
+        return workspaceFolders;
+    }
+
+    /**
+     * Set workspace folders.
+     *
+     * @param workspaceFolders List of workspace folders.
+     */
+    public void setWorkspaceFolders(List<WorkspaceFolder> workspaceFolders) {
```

Comment:
Since this is not implementing an interface here we can follow the Helidon style and omit the `set` prefix.